### PR TITLE
removed ignoring of non-json producing endpoints in Elm client

### PR DESCRIPTION
**Pull request by TeodoroFilippini**

### CODE REVIEW

File: swagger_to/elm_client.py [hunk 1]
```diff
@@ -600,7 +590,12 @@ def write_request(request: Request, fid: TextIO) -> None:
     types_str = ' -> '.join(types)
     types_str += ' -> '
     names_str = ' '.join(names)
-    line1 = function_name + ' : ' + types_str + 'Http.Request {}\n'.format(return_type)
+
+    line1 = function_name + ' : ' + types_str
+    if return_type:
```
Comment (mristin) [inline swagger_to/elm_client.py]: Please add a comparison operator here -- it makes the code a bit more robust (`is not None` or `return_type == ''`?).

Though this is considered unpythonic, I really find it difficult to distinguish the two cases and already experienced bugs due to it.

It's probably best to make:

```python3
return_type = None  # type: Optional[str]
...
```

File: tests/expected/elm/Client.elm [hunk 2]
```diff
@@ -411,6 +412,35 @@ updateMeRequest prefix maybeTimeout withCredentials updateUser =
         }
 
 
+{-| Contains a "get" request to the endpoint: /history, to be sent with Http.send
+    
+    The User Activity endpoint returns data about a user's lifetime activity with Uber. The response will
+    include pickup locations and times, dropoff locations and times, the distance of past requests, and
+    information about which products were requested.
+-}
+historyRequest : String -> Maybe Time.Time -> Bool -> Maybe Int -> Maybe Int -> Http.Request Activities
```
Comment (mristin) [inline tests/expected/elm/Client.elm]: The `Activities` seem to be a return type -- is this directly related to the feature change?

Could you maybe add a test case (a separate swagger def) that tests for non-JSON return type?

Comment (TeodoroFilippini) [reply tests/expected/elm/Client.elm]: I will add a test case for that!

**Comment by TeodoroFilippini** [comment]
@mristin here's the test!
